config/config.go

```diff
@@ -530,12 +530,13 @@ func DefaultFuzzConnConfig() *FuzzConnConfig {
 
 // MempoolConfig defines the configuration options for the Tendermint mempool
 type MempoolConfig struct {
-	RootDir   string `mapstructure:"home"`
-	Recheck   bool   `mapstructure:"recheck"`
-	Broadcast bool   `mapstructure:"broadcast"`
-	WalPath   string `mapstructure:"wal_dir"`
-	Size      int    `mapstructure:"size"`
-	CacheSize int    `mapstructure:"cache_size"`
+	RootDir          string `mapstructure:"home"`
+	Recheck          bool   `mapstructure:"recheck"`
+	Broadcast        bool   `mapstructure:"broadcast"`
+	WalPath          string `mapstructure:"wal_dir"`
+	MaxTxs           int    `mapstructure:"max_txs"`
+	MaxTxsTotalBytes int64  `mapstructure:"max_txs_total_bytes"`
```

Comment:
Maybe `MaxTxsBytes` is sufficient, since the `Total` is kind of redundant with the `s` on `Txs`. And Tendermint has no concept of `MaxTxBytes` or `MaxTxSingleBytes`, since we just bound txs by the block size instead (full block with 1 tx).

Actually, if we're going to keep `MaxTxs` as just `Size`, then maybe this is best as `SizeBytes` for now?

Reply:
> Actually, if we're going to keep `MaxTxs` as just `Size`, then maybe this is best as `SizeBytes` for now?

SizeBytes is too ambiguous. Does it include the cache size? Internal structures? ... MaxTxsBytes is much better